src/components/atoms/Avatar/index.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { useMedia } from "react-use";
+
+import { metricsSizes, styled, useTheme } from "@reearth/theme";
+
+import Text from "../Text";
+
+export type Props = {
+  size?: number;
+  avatar?: string;
+  color?: string;
+  userName?: string;
+  innerText?: string;
+};
+
+const Avatar: React.FC<Props> = ({ size = 24, avatar, color, userName, innerText }) => {
+  const theme = useTheme();
+  const isSmallWindow = useMedia("(max-width: 1024px)");
+  return (
+    <div>
+      <StyledAvatar size={size} avatar={avatar} color={color}>
+        {userName && (
+          <Text size={isSmallWindow ? "m" : "l"} color={theme.text.pale}>
+            {userName?.charAt(0).toUpperCase()}
```

Comment:
@lavalse design doesn't look like the letter needs to be uppercase. Unless he wants to change it to be like that, you can remove the `toUpperCase()`